README.md

```diff
@@ -0,0 +1,45 @@
+# aproxy - transparent proxy for HTTP and HTTPS/TLS
+
+Aproxy is a transparent proxy for HTTP and HTTPS/TLS connections. By pre-reading
+the Host header in HTTP requests and the SNI in TLS client hellos, it forwards
+HTTP proxy requests with the hostname, therefore, complies with HTTP proxies
+requiring destination hostname for auditing or access control.
+
+## Usage
```

Comment:
It'd be nice if there were another section like "Running from source" or "Developing" that explained how to run it locally without a Snap, with a couple of examples. `go run .` and any command line arguments necessary.

Reply:
Added, thanks!